src/prefect/client/client.py

```diff
@@ -941,6 +966,58 @@ def set_secret(self, name: str, value: Any) -> None:
         if not result.data.setSecret.success:
             raise ValueError("Setting secret failed.")
 
+    def get_task_tag_limit(self, tag: str) -> Optional[int]:
+        """
+        Retrieve the current task tag concurrency limit for a given tag.
+
+        Args:
+            - tag (str): the tag to update
+
+        Raises:
+            - ClientError: if the GraphQL query fails
+        """
+        query = {
+            "query": {
+                with_args("task_tag_limit", {"where": {"tag": {"_eq": tag}}}): {
+                    "limit": True
+                }
+            }
+        }
+
+        result = self.graphql(query)  # type: Any
+        if result.data.task_tag_limit:
+            return result.data.task_tag_limit[0].limit
+        else:
+            return None
+
+    def update_task_tag_limit(self, tag: str, limit: int) -> None:
+        """
+        Update the task tag concurrency limit for a given tag; requires tenant admin permissions.
+
+        Args:
+            - tag (str): the tag to update
+            - limit (int): the concurrency limit to enforce on the tag; should be a value >= 0
+
+        Raises:
+            - ClientError: if the GraphQL mutation is bad for any reason
+            - ValueError: if the tag limit-setting was unsuccessful, or if a bad limit was provided
+        """
+        if limit < 0:
+            raise ValueError("Concurrency limits must be >= 0")
+
+        mutation = {
+            "mutation($input: updateTaskTagLimitInput!)": {
```

Comment:
This GQL construction is more similar towhat I was suggesting above, where the complexity is encompassed entirely in a single `$input` variable